src/member/member.ts

```diff
@@ -22,14 +22,13 @@ export type MemberStorageItem = {
 };
 
 export type Pagination = {
-  totalItems: number;
-  totalPages: number;
-  currentPage: number;
+  totalCounts: number;
```

Comment:
I think this should be singular. 
```suggestion
  totalCount: number;
```